zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -303,11 +305,17 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
 
   $scope.runParagraph = function(data) {
```

Comment:
I will put numbers so that it's easy to follow the review.

1) There is too many runParagraph type of functions: `runparagraph`, `run`, `runParagraphAndMoveFocus`
I would advise to only use `$scope.run(boolean)`, move the `$scope.runParagraph` inside of it, and change all the `$scope.runParagraph` calls into `$scope.run`